packages/base/src/blockchainSchema.ts

```diff
@@ -0,0 +1,17 @@
+import { JsonObject, JsonValue } from "./jsonSchema";
+
+export type Hash = string;
+export type Amount = string;
+export type Address = Hash;
+export type UserId = Hash;
```

Comment:
Or should we just use `Address` instead?

Reply:
`UserId` is not hash for all blockchains, so just leaving it as `string` alias makes more sense. Same for `Address`.

ps:
Actually it looks like `UserId` doesn't fit very well to generic blockchain concept. Even for transfer where we use it for `from` -> `to` it's not always a user, but f.e. smart contract. So probably just leaving `Address` or renaming `User` to `Account` would make more sense